src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunctionParameterTemplate.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AppMagic.Authoring
 {
     internal sealed class ServiceFunctionParameterTemplate
     {
-        private readonly TypedName _typedName;
+        private TypedName _typedName;
```

Comment:
>TypedName [](http://example.com/codeflow?start=16&length=9)

Can we keep readonly?

Reply:
With SetTypedName, no.

We could return a new ServiceFunctionParameterTemplate in this method to keep readonly but that's going to be far less efficient and I don't see the benefit.

Reply:
There is no possible threading issue here but i can use the constructor to change that single value
Will be very inefficient but we could keep this field readonly